src/senaite/core/content/senaitesetup.py

```diff
@@ -83,3 +96,21 @@ def setEmailBodySamplePublication(self, value):
         """
         mutator = self.mutator("email_body_sample_publication")
         return mutator(self, value)
+
+    @security.protected(permissions.View)
+    def getEnableGlobalAuditlog(self):
+        """Returns if the global Auditlog is enabled
+        """
+        accessor = self.accessor("enable_global_auditlog")
+        return accessor(self)
+
+    @security.protected(permissions.ModifyPortalContent)
+    def setEnableGlobalAuditlog(self, value):
+        """Enable/Disable global Auditlogging
+        """
+        if value is False:
+            # clear the auditlog catalog
+            catalog = api.get_tool(AUDITLOG_CATALOG)
+            catalog.manage_catalogClear()
```

Comment:
I suggest to change the visibility of the "AuditLog" portal_action too, so the "Audit Log" item from the hamburger menu is only displayed when the audit log is enabled.

Reply:
Good point, I missed that one.
What do you think about leaving it all visible, but displaying a portal message in the Auditlog, telling the user that it is disabled and it can be reactivated in the setup?

Reply:
I like the idea!